### ISSUE TITLE
`fastp --help` and `fastp --version` : STDOUT versus STDERR

### ISSUE THREAD

**Issue by russHyde** [OpenGene/fastp#140]
Hi. 
I've just started using `fastp` and noticed that `fastp --help` and `fastp --version` direct their output to STDERR.
Could you confirm that this is the expected route for these messages, since it is more typical (eg, according to the GNU standards http://www.gnu.org/prep/standards/html_node/_002d_002dhelp.html and various implementations of argparse / optparse) for user-requested messages to be output to STDOUT.

Kind regards
Russ Hyde, U Glasgow

----
Ubuntu 18.04 using fastp via bioconda
`fastp --version`
`fastp 0.19.7`

**Comment by sfchen** [comment]
Will check it, thanks.

**Comment by tseemann** [comment]
I agree with @russHyde  - if I ask for `--help` it should be `stdout`, same with `--version` (and return 0).  However if you want to print the help because I did something wrong, it should be `stderr` (and return non-zero).

**Comment by ndokos** [comment]
Please fix this: it's a simple fix and it will make things easier for everybody. E.g. [look at the confusion of a new user who is trying to automate certain things using Org mode](https://emacs.stackexchange.com/questions/83543/run-code-blocks-output-format-conda-environment): his second problem is a combination of a syntax error on his part and the fact that `fastp --help` sends output to `stderr`, instead of `stdout`.
